app/assets/javascripts/components/student/student_contact.es6.jsx

```diff
@@ -31,16 +31,16 @@ class StudentContact extends Component {
     return (
         <div style={this.styles.container}>
           <h2 style={this.styles.heading}>{'Contact'}</h2>
-            {student.phone_number}
-            {student.home_address}
+            {student.cell_phone}
+            {student.home_phone}
             {student.email}
+            {student.home_address}
```

Comment:
And let's wrap all these attributes in `h5` tags: `<h5>{student.home_address}</h5>`.
We do this for consistency because all of these will take on the styles defined by our global heading styles.